pkg/testing/v1alpha1/domainmapping.go

```diff
@@ -18,27 +18,22 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	duckv1 "knative.dev/pkg/apis/duck/v1"
 	"knative.dev/serving/pkg/apis/serving/v1alpha1"
 )
 
 type domainMappingOption func(dm *v1alpha1.DomainMapping)
 
-type refOption func(*duckv1.KReference)
-
-func WithRef(namespace, name string, opt ...refOption) domainMappingOption {
+// WithRef fills reference field in domainmapping.
+func WithRef(namespace, name string) domainMappingOption {
```

Comment:
lint complaining here still, guess you need to capitalize domainMappingOption to make it happy 🤷

Reply:
oh you already did! heh